frontend/tet/shared/src/types/linkedevents.d.ts

```diff
@@ -18,6 +18,7 @@ export type LocalizedObject = {
 };
 
 export type IdObject = {
+  id: string;
```

Comment:
Would it be simpler to always use `@id` as the option value? Now they both required properties and there are compilation errors due to that.

Reply:
`id` could be set optional. To my understanding 'id' value from the requested keywords for the list is required (instead of `@id`) since linkedEvents API requires `yso:p4354` format for the event event filtering with keywords